tests/unit/test_utils.py

```diff
@@ -493,6 +493,104 @@ def test_tell_and_seek(self):
         self.assertEqual(chunk.tell(), 3)
         chunk.seek(0)
         self.assertEqual(chunk.tell(), 0)
+        chunk.seek(1, whence=1)
+        self.assertEqual(chunk.tell(), 1)
+        chunk.seek(-1, whence=1)
+        self.assertEqual(chunk.tell(), 0)
+        chunk.seek(-1, whence=2)
+        self.assertEqual(chunk.tell(), 2)
+
+    def test_tell_and_seek_boundaries(self):
+        data = b'thisstringis28characterslong'
```

Comment:
Instead of using this string, it would be nice if we could embed some of the expected functionality into the string value for example, if we make the string `abcd0123456789efgh` and have the start positing be at the character `0` and make the limits to the end of the numbers. It makes it easier to trace back the assertions, so for example it the test did a `tell` followed by a `read` it would be easy to understand why that content is expected:
```
        self.assertEqual(file_obj.tell(), 3)
        self.assertEqual(file_obj.read(), b'3456789')
```
And if we read outside of a boundary there would be a letter included in a read which would be wrong. I do not feel too strongly about this exact example value but something similar would be helpful in following the tests.